pyang/plugins/sid.py

```diff
@@ -640,13 +645,14 @@ def get_path(self, statement):
                         break
                     parent = parent.parent
 
-                if parent.i_module is None or parent.i_module != statement.i_module:
-                    path = "/" + statement.i_module.arg + ":" + statement.arg + path
-                else:
+                if (parent.i_orig_module is not None and parent.i_orig_module == statement.i_orig_module) or (parent.i_orig_module is None and prefix_module is not None and prefix_module == statement.i_orig_module):
```

Comment:
Please make it no longer than 80 characters per line

Reply:
Fixed